### PR TITLE
[MIG] Merge product_brand_*_report into product_brand

### PR DESCRIPTION
**Pull request by chienandalu**
* [product_brand_sale_report](https://github.com/OCA/sale-reporting/tree/9.0/product_brand_sale_report) can now be merged into `product_brand` since this one meet the dependency of `sale`.
* The same goes for [product_brand_invoice_report](https://github.com/OCA/account-invoice-reporting/tree/9.0/product_brand_invoice_report)
* I moved the readme to the new OCA's guidelines as well.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: product_brand/README.rst [hunk 1]
```diff
@@ -39,27 +58,31 @@ Known issues / Roadmap
...
 
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/product-attribute/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_brand%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/product-attribute/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/product-attribute/issues/new?body=module:%20product_brand%0Aversion:%2011.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (JayVora-SerpentCS) [inline product_brand/README.rst:67]: Keep text only upto feedback.

File: product_brand/__manifest__.py [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
     'name': 'Product Brand Manager',
-    'version': '11.0.1.0.2',
+    'version': '11.0.1.1.0',
     'category': 'Product',
     'summary': "Product Brand Manager",
-    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
-              ', MONK Software, Odoo Community Association (OCA)'
-              ', SerpentCS Pvt. Ltd.',
+    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA, '
+              'MONK Software, SerpentCS Pvt. Ltd., Tecnativa, '
```
Comment (JayVora-SerpentCS) [inline product_brand/__manifest__.py:15]: Make it Serpent Consulting Services Pvt. Ltd. (so its added to right author on search)

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: product_brand/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply product_brand/README.rst:67]: @JayVora-SerpentCS this text is now auto-generated by the script, so it's correct and self-mantained.

Comment (JayVora-SerpentCS) [reply product_brand/README.rst:67]: I see, I was referring to usual OCA readme format though.

Comment (pedrobaeza) [reply product_brand/README.rst:67]: This is the new OCA README format :wink:

You can see the new README in the template that states the automatic generation: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Review by rafaelbn — APPROVED** [review]
Tested functionally :+1: 

Please @JayVora-SerpentCS could unblock the merge? :smile:

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
Approved!

**Comment by pedrobaeza** [comment]
You need to make the migration part that merges module on OpenUpgrade, @chienandalu

**Pull request merged**